### PR TITLE
TECH-5427 Add new methods `read_client_connection` and `write_client_connection` to allow different connection settings for reads and writes.

### PR DESCRIPTION
**Pull request by ttstarck**

### CODE REVIEW

**Review by primiti — APPROVED** [review]
Nice and clean.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2021-03-23
```
Comment (alecjacobs5401) [inline CHANGELOG.md:7]: FWIW, we typically use `UNPUBLISHED` here as the date until we actually release the gem

File: lib/elasticsearch_models/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ElasticsearchModels
-  VERSION = "2.1.0"
+  VERSION = "2.2.0"
```
Comment (alecjacobs5401) [inline lib/elasticsearch_models/version.rb:4]: based on our [Gem Development](https://ringrevenue.atlassian.net/wiki/spaces/DEV/pages/750059580/Gem+Development) wiki, we should really be using patch versions during development and only do this "final" version when we are about to release.

E.g., this version should be 
```
2.2.0.pre.1
```

**Review by alecjacobs5401 — CHANGES_REQUESTED** [review]
Nice work! 

One thing - we should update the version to a patch version so we can properly test this in web before releasing a final version

File: CHANGELOG.md (hunk 1 above)
Comment (ttstarck) [reply CHANGELOG.md:7]: Kk, I'll leave this as it is and update it when we properly release the gem.

File: lib/elasticsearch_models/version.rb (hunk 2 above)
Comment (ColinDKelley) [reply lib/elasticsearch_models/version.rb:4]: Nit :"patch versions" -> "pre-release versions"

**Pull request merged**